src/apidefs/hello-world.api.ts

```diff
@@ -69,7 +69,7 @@ export const helloWorldApi = {
             name: 'userInfo',
             in: 'body',
             description: 'The user model instance to create.',
-            required: false,
+            required: true,
```

Comment:
Unrelated change

Reply:
I was trying to sneak this one in, but alas! I'll create another PR for it :-)